common/ferrostar/src/models.rs

```diff
@@ -193,6 +195,8 @@ pub struct RouteStep {
     pub geometry: Vec<GeographicCoordinate>,
     /// The distance, in meters, to travel along the route after the maneuver to reach the next step.
     pub distance: f64,
+    /// The duration, in seconds the router estimates it will take to travel the step.
```

Comment:
Minor wording adjustments; technically this may come from / be adjusted by other sources in the future, so we can generalize the wording a bit.

```suggestion
    /// The estimated duration, in seconds, that it will take to complete this step.
```